common/component.go

```diff
@@ -213,6 +217,17 @@ func (c *VcapComponent) Stop() {
 func (c *VcapComponent) ListenAndServe() {
 	hs := http.NewServeMux()
 
+	brokerConfig := c.Config.(*config.Config).Broker
+	serviceBroker := &f5servicebroker.F5ServiceBroker{&brokerConfig}
```

Comment:
Add comment temporarily using lager because of known issue with broker api however if fixed we should use our logger.